docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -869,7 +874,7 @@ connector.
 
    Performance tuning configuration properties are considered expert-level
    features. Altering these properties from their default values is likely to
-   cause instability and performance degradation. We strongly suggest that
+   cause instability and performance degradation. It is strongly suggested that
```

Comment:
I don't think we have a good standard for this .. maybe look up on google style .. but the passive voice here is probably not an improvement

Reply:
Will look up "we" in google style but "we" implies that the reader is also the subject, which they're not.

Reply:
No "we" https://developers.google.com/style/person

Reply:
Yeah... I know.. I dont like "we" and I know its wrong. Just that the passive voice is also kinda bad..